website/pages/docs/job-specification/hcl2/index.mdx

```diff
@@ -154,6 +154,17 @@ mounts = [
 ]
 ```
 
+Or better yet, the new [`mount`](/docs/driver/docker#mount) syntax is more appropriate here:
```

Comment:
Let's mention `mount` first with the disclaimer it is only supported >=1.0.1.

I'm afraid people will use whatever they run into first and never notice "Or better yet"